AppCenterData/AppCenterData/MSDataError.h

```diff
@@ -22,7 +22,7 @@
  *
  * @return A new `MSDataError` instance.
  */
-- (instancetype)initWithError:(NSError *)error;
+- (instancetype)initWithError:(NSError *)error; // TODO move to internal
```

Comment:
Created a bug item to move APIs marked as `// TODO move to internal` to internal headers.